src/components/Sidebar/ActionNode.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC, memo, useCallback, ChangeEvent } from 'react';
+import { SidebarButton } from '.';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+
+import DropDown from 'components/elements/DropDown';
+import { Pathway, State } from 'pathways-model';
+import useStyles from './styles';
+
+const nodeTypeOptions = [
+  { label: 'Action', value: 'action' },
+  { label: 'Branch', value: 'branch' }
+];
+
+interface ActionNodeProps {
+  pathway: Pathway;
+  currentNode: State;
+  changeNodeType: (event: string) => void;
+  addNode: (event: string) => void;
+}
+
+const ActionNode: FC<ActionNodeProps> = ({ pathway, currentNode, changeNodeType, addNode }) => {
+  const styles = useStyles();
+  const selectNodeType = useCallback(
+    (event: ChangeEvent<{ value: string }>): void => {
+      changeNodeType(event?.target.value || '');
+    },
+    [changeNodeType]
+  );
+
+  // The node has a key and is not the start node
+  const changeNodeTypeEnabled = currentNode.key !== undefined && currentNode.key !== 'Start';
+
+  // If the node does not have transitions it can be added to
+  const displayAddButtons = currentNode.key !== undefined && currentNode.transitions.length === 0;
+
+  return (
+    <>
+      {changeNodeTypeEnabled && (
+        <DropDown
+          id="nodeType"
+          label="Node Type"
+          options={nodeTypeOptions}
+          onChange={selectNodeType}
+          value="action"
+        />
+      )}
+      {displayAddButtons && (
+        <>
+          {changeNodeTypeEnabled && <hr className={styles.divider} />}
+          <SidebarButton
+            buttonName="Add Action Node"
+            buttonIcon={<FontAwesomeIcon icon={faPlus} />}
+            buttonText="Any clinical or workflow step which is not a decision."
```

Comment:
This text is a little unclear, and we should avoid the word decision because technically an action node does involve a decision to accept or decline. The idea behind an action node is that it's an atomic recommendation for treatment, for example a medication or a procedure.

Reply:
Good point. I think it would be helpful to get Jim's input for the text on all the buttons.